src/ontology/mondo-edit.obo

```diff
@@ -396823,6 +396825,7 @@ replaced_by: MONDO:0005073
 [Term]
 id: MONDO:0017415
 name: multiple pterygium syndrome
+def: "A syndrome that is characterized by permanently bent fingers, short stature, rocker-bottom or club feet, joints that are bent in a fixed position, union or webbing of the skin between the fingers, and/or webbing of the neck, inside bend of the elbows, back of the knees and armpits." [DOID:0080110]
 subset: disease_grouping
```

Comment:
@katiermullen I added this definition because I was trying to figure out the meaning of this term and why this was chosen as the superclass